src/web-auth/cross-origin-authentication.js

```diff
@@ -43,9 +43,12 @@ CrossOriginAuthentication.prototype.login = function(options, cb) {
   var _this = this;
   var theWindow = windowHelper.getWindow();
   var url = urljoin(this.baseOptions.rootUrl, '/co/authenticate');
+  options.username = options.username || options.email;
+  delete options.email;
```

Comment:
Why move the code up here? Just so it `email` can be deleted?

Reply:
yes